### DISCUSSION TITLE
Bounding box data for single entity

### DISCUSSION THREAD

**Discussion by noseeevil** [google/filament#4723] [Q&A]
hello! 

I can get bounding box for my **FilamentAsset* asset** via using **getBoundingBox()**:

Aabb bb_asset = asset->getBoundingBox();

but can I get something like that but for single entity inside asset?
I mean - bounding box data for single entity inside asset

**Comment by noseeevil** [comment]
I found some decision:

RenderableManager& rm = engine->getRenderableManager();

Instance renderableInstance = rm.getInstance(yourEntity); //utils::Entity e

Box singleBox = rm.getAxisAlignedBoundingBox(renderableInstance);

and now you can get information from **singleBox** - _singleBox.getMax()_, _singleBox.getMin()_ and etc

**Reply by MarxMc** [reply]
Is bounding box displayed over the 3d Object ?
